ospf6d/ospf6_top.h

```diff
@@ -38,6 +38,23 @@ enum {
 	OSPF6_LOG_ADJACENCY_DETAIL =	(1 << 1),
 };
 
+struct ospf6_redist {
+	unsigned short instance;
+	/* Redistribute metric info. */
+	struct {
+		int type;  /* External metric type (E1 or E2).  */
+		int value; /* Value for static metric (24-bit).
```

Comment:
I do not mind being forward thinking but break this up into it's own commit, if we are not using value right now.  Remember the goal here is to allow us reviewers to easily look at one thing at a time.  There should be a flow of commits that lead to the end goal if at all possible.  Instead we end up looking at things like value and why it's not being used.  Also shouldn't this be uint32_t if it's a 24 bit value?  Can it be negative?

Reply:
Only the structure is added similar to ospfv2. 
We will add the metric support for redistribute as part of another PR. 
Code changes are not too large to break into separate logical pieces. Therefore we can have a single commit.